### PR TITLE
Add Vue.js bindings

### PR DESCRIPTION
**Pull request by Kiho**
https://github.com/concretesolutions/redux-zero/issues/83

### CODE REVIEW

**Comment by matheusml** [comment]
This is awesome @Kiho 

Could you please:
- update the README file
- update the CHANGELOG

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "redux-zero",
-  "version": "4.5.2",
+  "version": "4.5.1",
```
Comment (matheusml) [inline package.json]: I should be 4.6.1

**Comment by Kiho** [comment]
@matheusml All done, Thanks.

**Comment by matheusml** [comment]
Thanks @Kiho 

Could you please resolve the conflicts?

**Comment by Kiho** [comment]
Hi @matheusml 

I forgot the link on readme, so I added it.  
Anything else is left here?

**Comment by matheusml** [comment]
Awesome job @Kiho 💯

**Pull request merged**